frontend/components/Navbar/index.tsx

```diff
@@ -16,17 +17,22 @@ export const Navbar = (): JSX.Element => {
   const SwitchIcon = useColorModeValue(FaMoon, FaSun);
   const nextMode = useColorModeValue("dark", "light");
 
+  const textColor = useColorModeValue("white", "white");
+  const boxBg = useColorModeValue("#343A40", "#343A40");
+  const buttonBg = useColorModeValue("#2D9CDB", "#2D9CDB");
+  const iconColor = useColorModeValue("#ECC94B", "light");
+
```

Comment:
Nice fix for this! we will probably need to implement a custom colorScheme so we can use the `LightMode` `DarkMode` components properly, but that can be done on a later date